image.yaml

```diff
@@ -37,45 +37,52 @@ modules:
                   ref: master
           - name: jboss-eap-modules
             git:
-                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
-                  ref: master
+                  url: https://github.com/wildfly/temp-eap-modules.git
+                  ref: EAP7-1216
           - name: jboss-eap-7-image
             git:
-                  url: https://github.com/jboss-container-images/jboss-eap-7-image.git
-                  ref: eap-cd
+                  url: https://github.com/wildfly/temp-eap-7-image.git
+                  ref: EAP7-1216
           - name: wildfly-cekit-modules
             git:
-                  url: https://github.com/kabir/wildfly-cekit-modules.git
+                  url: https://github.com/wildfly/wildfly-cekit-modules.git
                   ref: master
       install:
           - name: jboss.container.openjdk.jdk
             version: "8"
           - name: jboss.container.maven.35.bash
             version: "3.5scl"
-          - name: eap-cd-latest
+          - name: jboss.container.eap.cd
+            version: CD18
```

Comment:
Same comment here as https://github.com/wildfly/temp-eap-modules/pull/2

version: 18.0 matches the other versions we have, and .cd in the module name, namespaces it to CD